src/cli/openstudio_cli.rb

```diff
@@ -86,6 +86,31 @@
 
 module Gem
 class Specification < BasicSpecification
+
+  # This isn't ideal but there really is no available method to add specs for our use case. 
+  # Using self.dirs=() works for ruby official gems but since it appends the dir paths with 'specifications' it breaks for bundled gem specs 
+  def self.add_spec spec
+    warn "Gem::Specification.add_spec is deprecated and will be removed in RubyGems 3.0" unless Gem::Deprecate.skip
```

Comment:
This is not ideal but not sure what do here since this method is now remove in rubygems > 3.0. 
I think it's easy this or patching ruby source.